e2e/test/scenarios/embedding/embedding-native.cy.spec.js

```diff
@@ -112,7 +112,7 @@ describe("scenarios > embedding > native questions", () => {
       cy.findByText("CO");
       cy.findByText("Sid Mills").should("not.exist");
 
-      cy.location("search").should("eq", "?id=926&state=KS&product_id=10");
+      cy.location("search").should("eq", "?product_id=10&state=KS&id=926");
```

Comment:
What change we did that cause the parameter to change positions like this?

Reply:
it was because the order of returned by BE changed, I updated BE and it's reverted.

Reply:
Thanks for the explanation